vendor/vendor.json

```diff
@@ -3,6 +3,12 @@
 	"ignore": "test",
 	"package": [
 		{
+			"checksumSHA1": "+al5Z5nU/8wGul4Yo9WHdL6WocE=",
```

Comment:
yeah, we need this for web tokens or something - should have recognized it. going to blame it on low blood sugar